server/events/project_command_builder.go

```diff
@@ -401,5 +401,6 @@ func (p *DefaultProjectCommandBuilder) buildCtx(ctx *CommandContext,
 		User:              ctx.User,
 		Verbose:           verbose,
 		Workspace:         projCfg.Workspace,
+		Env:               map[string]string{},
```

Comment:
For me this is a signal that this field doesn't make sense in this struct

Reply:
I send it via argument